DallasTemperature.cpp

```diff
@@ -614,6 +614,15 @@ float DallasTemperature::rawToFahrenheit(int16_t raw) {
 
 }
 
+// Returns true if all bytes of scratchPad are '\0'
+bool DallasTemperature::isAllZeros(const uint8_t * const scratchPad, const size_t length) {
+	for (size_t i = 0; i < length; i++)

```

Comment:
My personal opinion, I prefer the style with open and curly braces. I find this can introduce annoying compilation bugs when you add new lines, for the sake of a couple of characters it's a good way to mitigate the paint.

Reply:
I've just tried to somehow adjust to an existing style. PR updated as you've suggested.